httpx/api.py

```diff
@@ -25,18 +25,16 @@ def request(
     headers: HeaderTypes = None,
     cookies: CookieTypes = None,
     auth: AuthTypes = None,
-    timeout: TimeoutTypes = None,
+    timeout: TimeoutTypes = DEFAULT_TIMEOUT_CONFIG,
     allow_redirects: bool = True,
     verify: VerifyTypes = True,
     cert: CertTypes = None,
     stream: bool = False,
-    trust_env: bool = None,
+    trust_env: bool = True,
 ) -> Response:
     """
     Sends an HTTP request.
-
```

Comment:
These spaces are critical, otherwise it gets mashed up into a single paragraph:
<img width="706" alt="Screen Shot 2019-11-26 at 10 25 44 AM" src="https://user-images.githubusercontent.com/1148665/69652604-8cd16400-1037-11ea-9151-1415ddd17e79.png">
vs.
<img width="705" alt="Screen Shot 2019-11-26 at 10 29 39 AM" src="https://user-images.githubusercontent.com/1148665/69652732-bd190280-1037-11ea-9c48-1e64bc9a8a8c.png">
...
<img width="703" alt="Screen Shot 2019-11-26 at 10 29 56 AM" src="https://user-images.githubusercontent.com/1148665/69652764-c6a26a80-1037-11ea-822a-a29ae738c600.png">

Reply:
Thanks! Not sure how those got mashed up.